docs/mesos/README.md

```diff
@@ -0,0 +1,87 @@
+<!--- master only -->
+> ![warning](../images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.18.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Mesos with Calico networking
+Calico can be used as a network plugin for Mesos both for the Docker
+Containerizer and the Unified Containerizer.
+
+### Docker Containerizer
+Calico with the Docker Containerizer uses a Calico network and IPAM 
+driver that hooks directly into the Docker networking infrastructure.
+Docker networks using the Calico plugin are created up-front, and Mesos 
+can then be used to launch Docker containers using these networks.  Each
+network is associated with a single Calico profile.  Fine grained policy
+can be modified using the `calicoctl profile` commands.
+
+### Unified Containerizer
+Calico with the Unified Containerizer uses the [Calico Mesos plugin]
+(https://github.com/projectcalico/calico-mesos) to configure
+networking for a Mesos agent that is using the [net-modules network
+isolator](https://github.com/mesosphere/net-modules). Networks are
+specified as net-groups when launching a Mesos task.  A Calico
+profile is automatically created for each net-group (if it doesn't
+already exist), defaulting to allow communication between containers
+using the same net-group.  Fine grained policy can be modified using
+the `calicoctl profile` commands.
+
+## Cluster Configuration Requirements
+The installation requirements to use Calico networking are different
```

Comment:
What point are you trying to get across here? It seems to me like this paragraph doesn't contain any important info